src/SprykerEco/Client/Easycredit/Dependency/Client/EasycreditToZedRequestClientInterface.php

```diff
@@ -14,7 +14,7 @@ interface EasycreditToZedRequestClientInterface
     /**
      * @param string $url
      * @param \Spryker\Shared\Kernel\Transfer\TransferInterface $object
-     * @param array|int|null $requestOptions Deprecated: Do not use "int" anymore, please use an array for requestOptions.
+     * @param array<mixed>|int|null $requestOptions Deprecated: Do not use "int" anymore, please use an array for requestOptions.
```

Comment:
According to this doc https://spryker.atlassian.net/wiki/spaces/CORE/pages/1395687852/CS+-+Docblocks+and+Collections you need to define key type too, e.g. array<string, mixed>